tests/src/test/scala/system/basic/WskActionTests.scala

```diff
@@ -266,8 +266,8 @@ class WskActionTests extends TestHelpers with WskTestHelpers with JsHelpers with
 
     val run = wsk.action.invoke(name, Map("payload" -> "google.com".toJson))
     withActivation(wsk.activation, run) { activation =>
-      activation.response.result shouldBe Some(
-        JsObject("stderr" -> "ping: icmp open socket: Operation not permitted\n".toJson, "stdout" -> "".toJson))
+      val stdout = activation.response.result.get.fields("stdout").convertTo[String]
+      stdout should not include ("bytes from")
```

Comment:
Good one! Should we also add should include("0 packets received") and should include("100% packet loss")?

Reply:
In our case the response is 

```
"response": {
    "result": {
      "stdout": "PING google.com (172.217.5.238): 56 data bytes\n",
      "stderr": "ping: permission denied (are you root?)\n"
    },
    "success": true,
    "status": "success"
  }
```

So due to permission check process probably gets killed and it cannot report further about stats

Reply:
Maybe checking the return code to be non-zero is worthwhile as well?

Reply:
Makes sense. Now capturing the process exit code and asserting against that as well